sidebar/window.c

```diff
@@ -288,8 +289,9 @@ static const char *sidebar_format_str(char *buf, size_t buflen, size_t col, int
   if (!m)
     return src;
 
-  bool c = Context && Context->mailbox &&
-           mutt_str_equal(Context->mailbox->realpath, m->realpath);
+  struct Mailbox *cm = ctx_mailbox(Context);
```

Comment:
Please can you rename this to `m_ctx`

Reply:
Mon Dec 21 13:22:10 GMT 2020

Done.  See 15bd1a7